bot.py

```diff
@@ -43,6 +44,53 @@ async def on_member_join(member):
     await member.dm_channel.send(f"Hi {member.name}, welcome to 관전남 월드!")
 
 
+# remove default help command to use our help command
+bot.remove_command("help")
+
+# Custom help command
+@bot.command(name="help", help="This is help command")
```

Comment:
the `help` string can be more descriptive instead of `This is help command`....
Of course we know `This is help command`...